modules/swagger-codegen/src/main/resources/python/swagger.mustache

```diff
@@ -199,8 +203,8 @@ class ApiClient(object):
     instance = objClass()
 
     for attr, attrType in instance.swaggerTypes.iteritems():
-        if obj is not None and attr in obj and type(obj) in [list, dict]:
-          value = obj[attr]
+        if obj is not None and instance.attributeMap[attr] in obj and type(obj) in [list, dict]:
+          value = obj[instance.attributeMap[attr]]
```

Comment:
In response, we should convert the json key to model attribute name.
As the json key in response is same with the key in model definition, we should convert it to attribute name.